src/jon_mikkel_ex/ex04/walker.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+
+__author__ = "Jon-Mikkel Korsvik"
+__email__ = "jonkors@nmbu.no"
+
+from random import getrandbits
+from pprint import pprint
+
+
+class Walker:
+    """
+    Walker class, with methods for checking position and moving.
+    Has its own method for finishing a trip home.
+
+    :param the_club: initial position of the walker
+    :param home: position of the walker's home
+    """
+    def __init__(self, the_club, home):
+        self.position = the_club
+        self.home = home
+        self.steps = 0
+
+    def move(self):
+        """
+        Uses a really efficient cointoss-  bool(getrandbits(1)) gives true or
+        false. Registers each move as a step.
+        :return: New position after move
+        """
+        if bool(getrandbits(1)):
```

Comment:
haha interesting! Also; protip: `poisiton += randint(0,1) * 2 -1`
In addition, casting to (transforming) a bool might not be necessary in python as it will evaluate `0-> False` and `1->True`:

```
In [18]: if 1: 
    ...:     print('ee') 
    ...:                                                                        
ee

In [19]: if 0: 
    ...:     print('ee') 
    ...:          
```